support-frontend/assets/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx

```diff
@@ -211,7 +199,7 @@ function updateTotal(props: PropTypes) {
 function getClickHandler(props: PropTypes, isCustomPrb: boolean) {
 	function onClick(event: StripePaymentRequestButtonElementClickEvent) {
 		trackComponentClick('apple-pay-clicked');
-		props.setAssociatedPaymentMethod();
+		props.setAssociatedPaymentMethod(Stripe);
```

Comment:
What is this change doing?

Reply:
Previously, on line 127, the `setAssociatedPaymentMethod` prop didn't take an argument but always passed `Stripe` to the `updatePaymentMethod` action creator; this just passes the argument directly given that the prop is now just a reference to `updatePaymentMethod`.

This should ultimately be removed as I'm pretty sure this is one of multiple non-serialisable values being put into Redux for the contributions checkout, but that's another large chunk of work.